utils/lib/solutions.ex

```diff
@@ -1026,11 +1026,17 @@ defmodule Utils.Solutions do
     defstruct []
   end
 
-  defprotocol BattleCharacter do
+  defprotocol Character do
+    @moduledoc File.read!(Path.expand("../exercises/battle_map.livemd"))
```

Comment:
Livebook is working on a feature to automatically run any doctests in a module, so we're leaning towards documenting modules in the exercise itself in anticipation of that feature arriving in Livebook. 

We're moving away from using the solutions module and the `utils` project entirely except for running scripts and tests. This way livebooks can be run without depending on the `utils` project.

Reply:
Very cool idea though reading the docs from the livemd and ensuring they remain in sync with the solution.

Reply:
Can you remove the character protocol from Solutions? Then this looks good to merge, well done!

Reply:
Having the tests run in Livebook would be great!